ogr/services/gitlab.py

```diff
@@ -221,7 +221,12 @@ def get_pr_list(self, status: PRStatus = PRStatus.open) -> List["PullRequest"]:
         return [self._pr_from_gitlab_object(mr) for mr in mrs]
 
     def get_sha_from_tag(self, tag_name: str) -> str:
-        raise NotImplementedError()
+        try:
+            tag = self.gitlab_repo.tags.get(tag_name)
+            return tag.attributes["commit"]["id"]
+        except gitlab.exceptions.GitlabGetError:
+            logger.error(f"Tag {tag_name} was not found.")
+            return ""
```

Comment:
Maybe it would be better to raise an exception?

Reply:
```suggestion
            raise PackitException(f"Tag {tag_name} was not found.", ex)
```

Reply:
Could you fix that also for github please?

Reply:
Why **PackitException**?

Reply:
Ups. `OgrException` (or ideally newly-created `GitlabAPIException`) would be of course better.